### PR TITLE
kk_KZ initial support

### PR DESCRIPTION
**Pull request by YerlenZhubangaliyev**
Good day,

I would like to add initial support for the Kazakh language

thank you

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -805,6 +805,40 @@ echo $faker->firstKanaName; // "ハルカ"
 echo $faker->lastKanaName; // "ナカジマ"
 ```
 
+### `Faker\Provider\kk_KZ\Company`
```
Comment (fzaninotto) [inline readme.md]: No need to document the formatted that exist in every language, like `name` or `company`. Please document here only the formatters specific to your locale.

File: src/Faker/Provider/kk_KZ/Person.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+{
+    protected static $maleNameFormats = array(
+        '{{lastName}}ұлы {{firstNameMale}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{lastName}}қызы {{firstNameFemale}}',
+    );
+
+    protected static $firstNameMale = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/kk_KZ/Person.php]: Could you add a link in the phpDoc to the source of this data? Same for other Person formatters

File: readme.md (hunk 1 above)
Comment (YerlenZhubangaliyev) [inline readme.md]: done

**Comment by fzaninotto** [comment]
Thanks!

**Comment by YerlenZhubangaliyev** [comment]
Thank you!

**Pull request merged**